query/common/src/main/java/org/codice/ddf/admin/common/fields/base/BaseEnumField.java

```diff
@@ -94,10 +94,13 @@ public void setValue(S value) {
   public List<ErrorMessage> validate() {
     List<ErrorMessage> validationMsgs = super.validate();
 
-    if (validationMsgs.isEmpty() && getValue() != null) {
-      if (getEnumValues().stream().map(EnumValue::getValue).noneMatch(v -> v.equals(getValue()))) {
-        validationMsgs.add(unsupportedEnum(getPath()));
-      }
+    if (validationMsgs.isEmpty()
```

Comment:
nit, the parentheses are different than the backport pr https://github.com/connexta/admin-console/pull/165/files#diff-948ec69e573b269f34c170b0731a7fbcR97